.github/workflows/cpp-post-commit.yaml

```diff
@@ -66,6 +66,7 @@ jobs:
         with:
           arch: ${{ inputs.arch }}
       - name: ${{ matrix.test-group.name }} tests
+        if: ${{ !(inputs.arch == 'grayskull' && matrix.test-group.name == 'ttnn ccl cpp unit tests') }}
```

Comment:
I'm not sure we should be doing this? I guess this is because the tests aren't intended to be running on GS and this  is to skip GS? Can we not make the tests just skip internally if on GS?